.github/workflows/run-tests.yml

```diff
@@ -98,44 +116,34 @@ jobs:
           echo "$(pwd)/bin" >> $GITHUB_PATH
         working-directory: neofs-node
 
-################################################################
-
-      - name: Download latest stable neofs-s3-gw
-        uses: dsaltares/fetch-gh-release-asset@1.1.1
-        with:
-          repo: 'nspcc-dev/neofs-s3-gw'
-          version: 'tags/v0.30.0'
-          file: 'neofs-s3-gw-linux-amd64'
-          target: 'neofs-testcases/neofs-s3-gw'
-
-      - name: Download latest stable neofs-s3-gw-authmate
-        uses: dsaltares/fetch-gh-release-asset@1.1.1
-        with:
-          repo: 'nspcc-dev/neofs-s3-gw'
-          version: 'tags/v0.30.0'
-          file: 'neofs-s3-authmate-linux-amd64'
-          target: 'neofs-testcases/neofs-s3-authmate'
+      - name: Build neofs-s3-gw
```

Comment:
You don't mention this change in the commit message.

Reply:
Split into different commits.